cmake/functions.cmake

```diff
@@ -242,8 +242,8 @@ function(add_app exe)
       add_executable(${exe} WIN32 ${sources} ${ALL_HEADERS})
     endif()
     add_windows_manifest(${exe})
-  elseif(OS_MACOSX)
-    add_executable(${exe} MACOSX_BUNDLE ${sources})
+  elseif(OS_MACOS)
+    add_executable(${exe} MACOS_BUNDLE ${sources})
```

Comment:
This should be reverted: https://cmake.org/cmake/help/v3.0/prop_tgt/MACOSX_BUNDLE.html

Reply:
Since this makes no practial difference right now (build fails long before this step), I will revert this in further pull requests.